docs/source/plugins/installation.md

```diff
@@ -0,0 +1,39 @@
+# Plugin Installation
+
+:::{index} Plugin Installation
+:::
+
+## Preinstalled Plugins
+Some plugins are built into and packaged with Arelle.
+These plugins are located in source under `/arelle/plugins`.
+No additional installation steps are necessary for these plugins.
+Activate pre-installed plugins [via the GUI using Select][gui-select] or [via CLI by name][cli-name].
+
+## Installing Plugins via Package Manager
+Published plugins can be installed by your Python package manager of choice.
```

Comment:
Should probably mention that this only works if you're running Arelle from source with your own Python environment (project:install.md#from-python-source).